content/en/platform/corda/4.10/enterprise/release-notes-enterprise.md

```diff
@@ -65,6 +65,12 @@ In this release:
 
 * The opentelemetry tracing signal is now supported in flows across nodes.
 
+* Previously, a memory leak in the transaction cache occurred due to weight of in-flight entries being undervalued. Improvements have been made to prevent in-flight entry weights from being undervalued and, because they are now estimated more correctly, this results in a large decrease in the total size of cached entities.
```

Comment:
```suggestion
* Previously, a memory leak in the transaction cache occurred due to weight of in-flight entries being undervalued. Improvements have been made to prevent in-flight entry weights from being undervalued. As they are now estimated more correctly, this results in a large decrease in the total size of cached entities.
```

Reply:
Done.